src/styles.css/blocks/card.css

```diff
@@ -0,0 +1,105 @@
+.card {
+  display: block;
+  padding: 16px;
+  width: 200px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+
+  margin: 16px auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.card__code {
+  margin-bottom: 16px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: #616070;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #616070;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.card__button {
+  width: 100%;
+  border: none;
+}
+
+.card__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 40px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+
+  text-transform: uppercase;
+  background-color: #00acdc;
+  border: 1px solid;
+  border-radius: 5px;
+}
+
+.card__link:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border-color: #00acdc;
```

Comment:
add this border-color directly to your button and you may not change this style on hover